Australian Frequency Plan Choice - AU915 (#593)
helium/HIP: 00YZ-au-frequency-plan-choice.md

# HIP YZ: Australian Frequency Plan Choice - AU915

- Author(s): [@plainsimpledot](https://github.com/plainsimpledot),
  [@Buckshot22](https://github.com/Buckshot22)(@Jules22), [@mcauser](https://github.com/mcauser),
  [@leogaggl](https://github.com/leogaggl), [@tonysmith55](https://github.com/tonysmith55)
- Start Date: 2023-02-16
- Category: Technical
- Original HIP PR:
- Tracking Issue:
- Status: Daft

## Summary

> This Helium Improvement Proposal is designed to allow the investors in the Helium network in Australia to have a voice in determining the country's best frequency plan, as originally proposed by the Helium Foundation in early 2022. The plan proposed by this HIP is AU915.

Reviewer: ```suggestion
This Helium Improvement Proposal is designed to allow the deployers of the Helium network in Australia to have a voice in determining the country's preferred frequency plan. The plan proposed by this HIP is AU915.
```

## Motivation

The LoRaWAN regional channel plan is critical to the operation of a LoRaWAN network. Two different channel plans are available for use in Australia, AU915 and AS923-1. The current Helium network in Australia operates on an AS923-1 channel plan variant. However, the community was not included in the decision-making process for the change in selecting this channel plan from the original AU915.

> The reasoning related to a footnote inserted in a LoRa Alliance document by an industry player, which the Alliance have confirmed in no way reflects the LoRa Alliance's official stance. The LoRa Alliance have made it very clear, and all of their official documentation confirms that they support both AU915 and AS923 in Australia, and the Alliance make no distinction between the two. The LoRa Alliance also confirmed that there is no intention to diminish the AU915 frequency plan in the future.

Reviewer: ```suggestion
The reasoning related to a footnote inserted in a LoRa Alliance document by an industry player, which some members of the Alliance have confirmed in no way reflects the LoRa Alliance's official stance. The LoRa Alliance have made it very clear, and all of their official documentation confirms that they support both AU915 and AS923 in Australia, and the Alliance make no distinction between the two. The LoRa Alliance also confirmed that there is no intention to diminish the AU915 frequency plan in the future at the time this HIP was written.
```

Reviewer 2: Have modified this one slightly

> We believe, backed up by expert opinions including senior members of the LoRa Alliance, that the only effective way to support AS923 is to utilise 16-channel gateways, dual-head 8-channel gateways, or allow investors/operators in densely populated areas to voluntarily operate on AS923 in line with the Helium IOT PoC roadmap goal of reducing congestion.

Reviewer: ```suggestion
We believe, backed up by expert opinions including senior members of the LoRa Alliance, that the only effective way to support AS923 is to utilise 16-channel gateways, dual-head 8-channel gateways, or allow operators in densely populated areas to voluntarily operate on AS923 in line with the Helium IOT PoC roadmap goal of reducing congestion.
```

AU915 is better suited to the Australian environment/use cases due to

1. The availability of higher power sensors (as the default configuration)
2. The higher likelihood of roaming opportunities and partnerships in the public AU915 ecosystem is likely to bring more significant benefits than a single commercial operator who has limited use cases and provides coverage in areas where they are utilising sensors.

> 3. AU915 also allows gateway operators who will move over to alternative networks the potential to dual-purpose their gateways and sensors rather than lose them from the Helium network completely. Operators in other parts of the world currently have and are utilising this option where the current situation in Australia creates competing networks, which in a market with low population density and the geography of Australia will see all parties lose out.

Reviewer: ```suggestion
```
I would remove dual mining as a reason for this HIP. Given the MCC's concerns with dual mining, it weakens the argument.

Reviewer 2: Ack

6. Australia is in a unique position because it has two valid choices fully supported by the LoRa Alliance. Allowing the Australian community the opportunity to decide on their future will restore a lot of faith in the Helium ecosystem that has been lost over the past six months.

The proposed change is extremely simple in reverting to a known working configuration (AU915 FSB2) utilising a LoRa Alliance-approved frequency plan at up to 30 dBm EIRP which is within the permitted range for Australia.

It is vital for the Community of Helium operators to participate in the decision-making process. The previous lack of Community participation in the decision-making process has led to the need for this proposal. It is essential to recognise that not allowing the Community to participate in the decision-making process may lead to significant risks, such as a lack of community adoption, mistrust, and a loss of community engagement.

Further risks in not switching to AU915 are that the current trend downwards in both operating gateways and data usage will continue to show the dramatic decline it has shown since the deployment of the
'dual plan'

### Australia Usage

> ![Australia Usage 20 weeks](00YZ-au-frequency-plan-choice/helium_usage_AU_2023-03-07.png)

Reviewer: happy to provide some updated charts here if needed.

### United States Usage

> ![Australia Usage 20 weeks](00YZ-au-frequency-plan-choice/helium_usage_US_2023-03-07.png)

Reviewer: ```suggestion
![United States Usage 20 weeks](00YZ-au-frequency-plan-choice/helium_usage_US_2023-03-07.png)
```

### Worldwide Usage

> ![Australia Usage 20 weeks](00YZ-au-frequency-plan-choice/helium_usage_WORLD_2023-03-07.png)

Reviewer: ```suggestion
![Worldwide Usage 20 weeks](00YZ-au-frequency-plan-choice/helium_usage_WORLD_2023-03-07.png)
```

## Rationale and Alternatives

Australia has two valid frequency plans available under local laws and supported by the LoRa Alliance. The Alliance make no official distinction between the two and both are freely available for use.

The availability of gateways and sensors suitable for a particular regional plan must be considered. Research indicates that both regional plans have supply chain support from Manufacturers making the selection seem less critical in this regard, however only AU915 has availability for the higher power sensors which are critical for many use cases in Australia. Further, due to the complication with needing to configure AU915 sensors to operate under the AU923-1 configuration, there is significant additional work required to accomplish this. This is detracting from the IoT experience placing at risk the speed of IoT adoption and slowing the expansion of network coverage.

This HIP simply provides the Australian community with the ability to choose the frequency plan to be used in the country, weighing up all of the factors and the changing landscape with the public AU915 networks.

> Doing so brings the decision in line with the Helium governance principles and the advertised "People's Network". As has been outlined above, a Yes vote does not preclude future support for AS923, however, such support should be introduced in a manner that does not diminish full AU915 support to allow roaming from all AU915 networks, while introducing support for AS923 using more capable hardware.

Reviewer: ```suggestion
As has been outlined above, a Yes vote does not preclude future support for AS923, however, such support should be introduced in a manner that does not diminish full AU915 support to allow roaming from all AU915 networks, while introducing support for AS923 using more capable hardware.
```
remove editorializing.

"Full Support" in this context means that any device configured for the frequency plan in question will be able to operate on the network and other networks on the same frequency plan without any requirement for configuration changes.

The public networks currently and planned for Australia, all operating on AU915, are much more likely to provide roaming opportunities than the commercial operators who are much more likely to, and usually, deploy sensors in conjunction with their own gateway infrastructure.

## Voting mechanism

> It is proposed to use the standard Helium on-chain voting system [HeliumVotes](https://heliumvote.com) to allow any wallet to vote on this HIP. See [Voting on Helium](https://docs.helium.com/community-voting/) documentation on Helium Docs.

Reviewer: ```suggestion
It is proposed to use the standard Helium on-chain voting system [Helium Vote](https://heliumvote.com) to allow any wallet to vote on this HIP. See [Voting on Helium](https://docs.helium.com/community-voting/) documentation on Helium Docs.
```

2. Device Compatibility: increased device compatibility is measured by monitoring the number of devices that are now able to connect to the network following the selection of the channel plan. An increase in the number of compatible devices would indicate a successful selection of the channel plan and increased device compatibility.
3. Adoption Rate: monitoring the adoption rate of the network in the region by tracking the number of new users and applications developed on the network following the selection of the channel plan. An increase in the adoption rate would indicate a successful selection of the channel plan and increased adoption of the network by businesses and organisations in the region.
4. Improved Service Quality: monitoring the quality of service provided by the network by tracking the number of network failures or disruptions following the selection of the channel plan. A reduction in these failures or disruptions would indicate a successful selection of the channel plan and improved service quality for users of the network.

> 5. Restore faith in Helium: We expect this change would restore confidence in investors who purchased Helium-related hardware and have suffered reduced performance since November. We expect a restoration of faith in the Helium governance system and less of a feeling that the Australian investors have been ignored.

Reviewer: ```suggestion
5. Restore faith in Helium: We expect this change would restore confidence in Hotspot owners who purchased Helium-related hardware and have suffered reduced performance since November. We expect a restoration of faith in the Helium governance system and less of a feeling that the Australian users have been ignored.
```
